### DISCUSSION TITLE
"Let it Drain" talent should have a counterpart

### DISCUSSION THREAD

**Discussion by Prolbo** [FakeFishGames/Barotrauma#11399] [Ideas and suggestions]
Perk "Let it Drain" should have a counterpart / alternative because in cases when there is more than one mechanic in the crew it becomes not just useless but actively harmful. 

How? By demanding skill point with literally NO gain AND at the same time increasing expirience progression counter for the character.

I propose to add thematically named "When it Rains" alternative perk which should provide bonus swimming speed inside the submarine and allow to manually power up mechanical systems on the submarine. Manual powering (on top of requiring effort and constant presence of mechanic) should be less efficient and damage machine in case of low mechancal skill. If you feel generous you can allow manual powering of electrical systems with harsher penalties.

**Comment by NoIWontBruh0831** [comment]
New Mechanic Machinist talent:

"Let it Rain". Counterpart to "Let it Drain"
Effect: 
> Inside the submarine you swim 40% faster. Unlocks recipe Handcrank

Hankcrank effect:
> - Equip the Handcrank and press E on a mechanical/electrical system. This makes you spin the crank in circles, powering up the mechanical or electrical system while damaging you AND the system in the process. Complete small quicktime events to increase power generated. 
> - Using the Handcrank on a system that still has electricity will instead overclock the system, increasing its strength but will damage it even more.
> - Requires 90 Mechanical/Electrical for max efficiency and, even so, will still damage you and the system.

Use this item to force Engines, Deconstructors, Pumps, etc to work while flooded and/or with no power. You can also temporarily overclock those systems for increased output

---

Crafting recipe:
> No idea lol


I don't know if this is what you had in mind when making your suggestion but I tried to make it into an item lol

**Reply by MarkCringan** [reply]
Great idea but I'd say just eliminate the handcraft and allow them to do what the old tinkering talent did, but maybe at only 20-50% efficiency. None of the old tripling the efficiency BS.
